packages/mongo/mongo_driver.js

```diff
@@ -992,6 +992,13 @@ MongoConnection.prototype._createSynchronousCursor = function(
     replaceTypes(cursorDescription.selector, replaceMeteorAtomWithMongo),
     cursorOptions.fields, mongoOptions);
 
+  if (typeof cursorOptions.maxTimeMS !== 'undefined') {
```

Comment:
Hi @bpartridge - any progress on the requested changes? Let me know if you're busy and I'll finish things up. Thanks!